src/balancereader/src/test/java/anthos/samples/bankofanthos/balancereader/BalanceReaderControllerTest.java

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright 2020, Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package anthos.samples.bankofanthos.balancereader;
+
+import java.util.concurrent.ExecutionException;
+
+import com.auth0.jwt.JWTVerifier;
+import com.auth0.jwt.exceptions.JWTVerificationException;
+import com.auth0.jwt.interfaces.Claim;
+import com.auth0.jwt.interfaces.DecodedJWT;
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.LoadingCache;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mock;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.*;
+import static org.mockito.MockitoAnnotations.initMocks;
+
+class BalanceReaderControllerTest {
+
+    private BalanceReaderController balanceReaderController;
+
+    @Mock
+    private JWTVerifier verifier;
+    @Mock
+    private LedgerReader ledgerReader;
+    @Mock
+    private DecodedJWT jwt;
+    @Mock
+    private Claim claim;
+    @Mock
+    private TransactionRepository transactionRepository;
+    @Mock
+    private LoadingCache<String, Long> cache;
+
+    private static final String VERSION = "v0.2.0";
+    private static final String LOCAL_ROUTING_NUM = "123456789";
+    private static final int CACHE_SIZE = 100000;
```

Comment:
where does this get used?

Reply:
Ah, forgot to take it out after I refactored the BalanceCache